packages/teuchos/core/test/MemoryManagement/ArrayView_UnitTests.cpp

```diff
@@ -334,6 +350,7 @@ TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( ArrayView, danglingView_rcp_std_vector, T )
   }
 #ifdef TEUCHOS_DEBUG
   TEST_THROW(av.getRawPtr(), DanglingReferenceError);
+  TEST_THROW(av.data(), DanglingReferenceError);
```

Comment:
@bartlettroscoe Here's the unit test that `.data()` does the same debug-mode dangling reference testing that `.getRawPtr()` does.

Reply:
Okay, I missed that.  Thanks for adding that!   I think we are 100% good then.